installer.php

```diff
@@ -513,11 +513,13 @@ function install(): void
     <br />
     This is the cron info you need for section <b>1.2 Cronjobs</b> of the installation instructions.<br />
     <pre>
-    */5 * * * * php $path/cron_fivemins.php $code
-    * * * * * php $path/cron_minute.php $code
-    0 * * * * php $path/cron_hour.php $code
-    0 0 * * * php $path/cron_day.php $code
-    </pre>
+    * * * * * php $path/crons/CronHandler.php cron=minute-1 code=$code
+    */5 * * * * php $path/crons/CronHandler.php cron=minute-5 code=$code
+    0 * * * * php $path/crons/CronHandler.php cron=hour-1 code=$code
+    0 0 * * * php $path/crons/CronHandler.php cron=day-1 code=$code
+    </pre><br>
+    Alternatively, you can toggle the \"Use Timestamps Instead of Cron Jobs\" option in the Basic Settings on the Staff Panel to use timestamps instead.<br><br>
```

Comment:
Is there a performance hit to whichever user runs this as part of request?

Reply:
There will likely be an exponential performance hit as the database grows bigger.
This method was added more as a stop-gap measure for a lack of cron job support on a per-(local)host situation.

Production-ready games with cron support are advised to make use of said cron jobs and not rely on this in the long-term.